zstacklib/zstacklib/utils/pci.py

```
@@ -213,3 +213,31 @@ def get_pci_device_ids():
+            dependent_addr = item.replace("consumer:pci:", "")
+            pcis.append(dependent_addr)
+    return pcis
+
+
+@linux.retry(3, 3)
+def find_pci_device(vm_uuid, pci_addr):
+    domain, bus, slot, function = parse_pci_device_address(pci_addr)
+    cmd = """virsh dumpxml %s | grep -A3 -E '<hostdev.*pci' | grep 
+    "<address domain='0x%s' bus='0x%s' slot='0x%s' function='0x%s'/>" """ % \
+          (vm_uuid, domain, bus, slot, function)
+    r, o, e = bash_roe(cmd)
```

Ruff (0.14.3)

Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)